### PR TITLE
Add --all option to `light rm` command to remove all peers

### PR DESCRIPTION
**Pull request by romac**
Closes: #431

## Description

Add `--all` option to `light rm` command to remove all peers for a given chain.

______

For contributor use:

- [x] Updated the __Unreleased__ section of [CHANGELOG.md](https://github.com/informalsystems/ibc-rs/blob/master/CHANGELOG.md) with the issue.
- [ ] If applicable: Unit tests written, added test to CI.
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Updated relevant documentation (`docs/`) and code comments.
- [ ] Re-reviewed `Files changed` in the Github PR explorer.

### CODE REVIEW

File: relayer-cli/src/commands/light/rm.rs [hunk 1]
```diff
@@ -1,35 +1,42 @@
...
+    /// identifier of the chain to remove peers from
     chain_id: ChainId,
 
-    /// peer id for this client
-    peer_id: PeerId,
+    /// identifiers of peers to remove
+    peer_ids: Vec<PeerId>,
+
+    /// remove all peers, implies --force
+    all: bool,
```
Comment (ancazamfir) [inline relayer-cli/src/commands/light/rm.rs:39]: Can we have a flag to skip confirmation?

Comment (romac) [reply relayer-cli/src/commands/light/rm.rs:39]: Right, I forgot about that, will get on it!

Comment (romac) [reply relayer-cli/src/commands/light/rm.rs:42]: Done

**Review by adizere — APPROVED** [review]
Is this ready? Looks good to me!

**Review by ancazamfir — APPROVED** [review]
Thanks Romain!

**Pull request merged**